dynawo/sources/Modeler/DataInterface/IIDM/DYNBatteryInterfaceIIDM.cpp

```diff
@@ -98,7 +98,8 @@ BatteryInterfaceIIDM::importStaticParameters() {
   staticParameters_.insert(std::make_pair("qMin", StaticParameter("qMin", StaticParameter::DOUBLE).setValue(qMin)));
   staticParameters_.insert(std::make_pair("targetP_pu", StaticParameter("targetP_pu", StaticParameter::DOUBLE).setValue(getTargetP() / SNREF)));
   staticParameters_.insert(std::make_pair("targetP", StaticParameter("targetP", StaticParameter::DOUBLE).setValue(getTargetP())));
-  double sNom = sqrt(pMax * pMax + qMax * qMax);
+  double qNom = getQNom();
+  double sNom = sqrt(pMax * pMax + qNom * qNom);
```

Comment:
Thanks ! (nrt to be checked though, I suppose)

Reply:
already did that, everything looks ok :)